_posts/2024-04-29-dsaa-eras/dsaa-eras.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "DSAA Eras"
+description: |
+  Our experiences with data science in the hospital.
+author:
+  - name: Chloe Pou-Prom on behalf of DSAA
+    url: {}
+date: 2024-04-29
+output:
+  distill::distill_article:
+    toc: true
+categories:
+  - project-ed-forecasting
+  - project-chartwatch
+  - project-ops-center
+  - post-miscellaneous
+draft: true
+---
```

Comment:
I want to create an Eras-style poster as the preview image lol: https://www.myerastourposter.com/create

Reply:
love this idea!!